---
title: "&#39;Infine&#39; non può trovarsi all'esterno di un &#39;provare a&#39; istruzione"
ms.date: 07/20/2015
f1_keywords:
- vbc30382
- bc30382
helpviewer_keywords:
- BC30382
ms.assetid: 0314d8d2-18bc-4bbd-858c-0a18408b52fd
ms.openlocfilehash: 0e16604aed715884df7ca4aa52716804603b7533
ms.sourcegitcommit: 3d5d33f384eeba41b2dff79d096f47ccc8d8f03d
ms.translationtype: MT
ms.contentlocale: it-IT
ms.lasthandoff: 05/04/2018
ms.locfileid: "33613188"
---
# <a name="39finally39-cannot-appear-outside-a-39try39-statement"></a>&#39;Infine&#39; non può trovarsi all'esterno di un &#39;provare a&#39; istruzione
`Finally` viene usato per completare un blocco `Try...Catch...Finally` , quindi può essere presente solo una volta alla fine del blocco. È presente un'istruzione `Finally`non necessaria oppure l'istruzione `Finally` si trova al di fuori dei limiti del blocco `Try` corrispondente.  
  
 **ID errore:** BC30382  
  
## <a name="to-correct-this-error"></a>Per correggere l'errore  
  
1.  Individuare e rimuovere il `Finally` istruzione.  
  
2.  Spostare l'istruzione `Finally` nella posizione appropriata nel codice.  
  
## <a name="see-also"></a>Vedere anche  
 [Istruzione Try...Catch...Finally](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)  
 
